projects/aca-folder-rules/src/lib/manage-rules/manage-rules.smart-component.html

```diff
@@ -26,13 +26,24 @@
                               class="aca-manage-rules__actions-bar__title__breadcrumb"></adf-breadcrumb>
             </adf-toolbar-title>
 
-            <button
-              *ngIf="canEditRule(mainRuleSet$ | async)"
-              data-automation-id="manage-rules-create-button"
-              mat-flat-button color="primary"
-              (click)="openCreateUpdateRuleDialog()">
-              {{ 'ACA_FOLDER_RULES.MANAGE_RULES.TOOLBAR.ACTIONS.CREATE_RULE' | translate }}
-            </button>
+            <mat-slide-toggle
+              data-automation-id="manage-rules-inheritance-toggle-button"
+              [checked]="isInheritanceEnabled"
+              (change)="onInheritanceToggleChange($event)"
+              [disabled]="isInheritanceToggleDisabled"
+              [labelPosition]="'before'">
+              {{ 'ACA_FOLDER_RULES.MANAGE_RULES.TOOLBAR.ACTIONS.INHERIT_RULES' | translate }}
+            </mat-slide-toggle>
+
+            <ng-container *ngIf="canEditRule(mainRuleSet$ | async)">
+              <mat-divider vertical [style]="{height:'50%', margin:'0 12px'}"></mat-divider>
```

Comment:
Can we move inline styles to some stylesheet?

Reply:
Moved